### ISSUE TITLE
Can you please add an autocomplete parameter for combobox, like true or false, in order to search through the list of items?

### ISSUE THREAD

**Issue by Alefrasca90** [TomSchimansky/CustomTkinter#1654]

**Comment by Akascape** [comment]
@Alefrasca90
Try this new dropdown menu. It has a autocomplete feature:
https://github.com/Akascape/CTkScrollableDropdown

**Comment by Alefrasca90** [comment]
I tried it, but in my app I have 40 of them automatically created, this causes an issue where the root pops up every time making the app taking so much time to create

**Comment by ElectricCandlelight** [comment]
You have 40 combo boxes on screen at one time? Is there not a way to simplify that? because that sounds like a UI/UX nightmare.

**Comment by Akascape** [comment]
@ElectricCandlelight 
There is no solution in that case, I have checked replacing the top level class, it takes the same time to load 40 boxes (almost 7-9s).
You either have to simplify the UI, or implement the autocomple feature in the tkmenu of combobox.

**Comment by Alefrasca90** [comment]
Yes, thanks, I need it to be fast because I'll run the app on a raspberry that doesn't have a lot of performances.
